### DISCUSSION TITLE
Directional Flak Cannon Ammo Box is still Two-Handed

### DISCUSSION THREAD

**Discussion by UnhappyWithMyUsername** [FakeFishGames/Barotrauma#14162] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

As title says, it still uses the old code for the <Holdable> component and thus is not combinable, still carried with both hands and doesn't slow you down:
`<Holdable slots="RightHand+LeftHand" holdpos="0,-80" handle1="-40,30" handle2="40,30" aimable="false" msg="ItemMsgPickUpSelect"/>`

When it should be:
```
<Holdable canbecombined="true" removeoncombined="false" slots="RightHand,LeftHand" holdpos="0,-30" handle1="0,-30" aimable="false" msg="ItemMsgPickUpSelect">
  <StatValue stattype="MovementSpeed" value="-0.2" />
</Holdable>
```

### Reproduction steps

1. Get a directional flak ammo box.
2. Pick it up and witness the issue.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.7.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! This has been fixed in the latest hotfix, closing.
